recipes-core/niacctbase/niacctbase.bb

```diff
@@ -27,17 +28,19 @@ GROUPADD_PARAM_${PN} = " --system ${LVRT_GROUP}; \
 USERADD_WEBSERVICES = "-m -N -g ${LVRT_GROUP} -G niwscerts,plugdev,adm,tty -c 'Web services user' webserv"
 
 # add parameter -m if you want home directories created with default files (.profile, .bashrc)
-USERADD_PARAM_${PN} = " -m -N -g ${LVRT_GROUP} -G network,niwscerts,plugdev,tty,video -c 'LabVIEW user' ${LVRT_USER}; \
-		-N -g openvpn -G network -c 'OpenVPN' -r openvpn; \
-		${@oe.utils.conditional('DISTRO', 'nilrt-nxg', '', d.getVar('USERADD_WEBSERVICES', True), d)}"
+USERADD_PARAM_${PN} = " \
+	-m -N -g ${LVRT_GROUP} -G network,niwscerts,plugdev,tty,video -c 'LabVIEW user' ${LVRT_USER}; \
+	-N -g openvpn -G network -c 'OpenVPN' -r openvpn; \
+	${@oe.utils.conditional('DISTRO', 'nilrt-nxg', '', d.getVar('USERADD_WEBSERVICES', True), d)} \
+"
```

Comment:
Fun fact: none of these settings actually get applied, because they are overwritten by a static assignment by the `useradd-staticids` class. I presume these were added for `nilrt-nxg`, but are now vestigial.

I didn't consider it within the scope of the work that I'm doing to track this down, but hopefully someone will remember this note later.